predict.py

```diff
@@ -1,55 +1,31 @@
 #! /usr/bin/env python
 
-import argparse
+import cli
 import os
 import cv2
 import numpy as np
 from tqdm import tqdm
-from preprocessing import parse_annotation
 from utils import draw_boxes
 from frontend import YOLO
-import json
 
 os.environ["CUDA_DEVICE_ORDER"] = "PCI_BUS_ID"
 os.environ["CUDA_VISIBLE_DEVICES"] = "0"
 
-argparser = argparse.ArgumentParser(
-    description='Train and validate YOLO_v2 model on any dataset')
 
-argparser.add_argument(
-    '-c',
-    '--conf',
-    help='path to configuration file')
+def _main_(argstate):
```

Comment:
Shouldn't be using underscores in the main method name (I know you didn't write this, but may as well get rid of them since it's a bit weird).

Reply:
Good idea.

Reply:
Do you think we should do this in a separate pull request?